src/uct/tcp/tcp_sockcm.h

```diff
@@ -7,30 +7,39 @@
 #include <uct/base/uct_cm.h>
 
 
+typedef struct uct_tcp_sockcm_ep   uct_tcp_sockcm_ep_t;
+
+
 typedef struct uct_tcp_sa_arg {
-    int             fd;
-    ucs_list_link_t list;
+    int                 fd;
+    ucs_list_link_t     list;
+    uct_tcp_sockcm_ep_t *ep;
 } uct_tcp_sa_arg_t;
 
 
 /**
  * A TCP connection manager
  */
 typedef struct uct_tcp_sockcm {
-    uct_cm_t        super;
-    size_t          priv_data_len;
+    uct_cm_t            super;
+    size_t              priv_data_len;
     /** List of open file descriptors. One per client */
```

Comment:
if `fd` is not needed anymore in the `uct_tcp_sa_arg_t` strcture, pls fix the comment here:
```
/** List of EPs. One per client */
```